site/content/overview/gateway-api-compatibility.md

```diff
@@ -178,6 +178,7 @@ See the [static-mode]({{< relref "/reference/cli-help.md#static-mode">}}) comman
       - `ResolvedRefs/False/RefNotPermitted`
       - `ResolvedRefs/False/BackendNotFound`
       - `ResolvedRefs/False/UnsupportedValue`: Custom reason for when one of the HTTPRoute rules has a backendRef with an unsupported value.
+      - `ResolvedRefs/False/InvalidIPFamily`: Custom reason for when one of the HTTPRoute rules has a backendRef that has unsupported IPFamily.
```

Comment:
I would say "mismatched" or "invalid" IPFamily, not unsupported. Makes it sound like we should just use UnsupportedValue using that wording.